src/main/resources/edu/cornell/kfs/module/ld/cu-spring-ld.xml

```diff
@@ -95,5 +95,16 @@
    <import resource="document/validation/configuration/SalaryExpenseTransferValidations.xml" />
    <import resource="document/validation/configuration/CuYearEndBenefitExpenseTransferValidations.xml" />
    <import resource="document/validation/configuration/CuYearEndSalaryExpenseTransferValidations.xml" />
+   
+   <!-- CU Customization KFSPTS-34828 
+   partial backport of FINP-10395 to fix opening general ledger from salary expense transfer document -->
+   <bean id="laborLedgerEntryForExpenseTransferLookupableHelperService"
```

Comment:
It looks like FINP-10395 only modified the parent bean. Can the override of this child bean be removed?

Reply:
you're right, not needed, I shall remove